ComputerVision/CIFAR/scripts/launch_pretraining.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+seed=${seed:-0}
+lr=${learning_rate:-"0.1"}
+model=${model:-"resnet_18"}
+data=${data:-"cifar100"}
+epochs=${epochs:-"200"}
+batch_size=${batch_size:-"128"}
+log_interval=${log_interval:-"100"}
+weight_decay=${weight_decay:-"5e-4"}
+output_dir=${output_dir:-"/cb/home/vithu/extra-storage/want/resnet18"}
```

Comment:
remove traces of `/cb/home/vithu` ? and just point to `output_dir/resnet18` or something like that ?